### PR TITLE
Feature/Add next steps page to gameplay UX.

### PR DESCRIPTION
**Pull request by rhiannonstanford**
Add placeholder page to complete gameplay route.  

Add directory NextSteps to pages.
- add index.js
- NextStepsContainer.js
- RenderNextSteps.js

Reorder gameplay imports in app index.js to be ordered by UX.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -53,12 +53,12 @@ import FaceoffReveal from './components/pages/Animations/FaceoffReveal';
 
 // GamePlay Components
 import { GamePlayMain } from './components/pages/GamePlay/index';
```
Comment (rhiannonstanford) [inline src/index.js:55]: Reordered gameplay imports to be consistent with UX order.

**Review by rhiannonstanford — COMMENTED** [review]
Self review.

**Review by Yonathan-Admasu728 — APPROVED** [review]
great fixes in all 4 files. Thank you!

**Review by ethankarp — APPROVED** [review]
Looks good to me, nice job

**Review by OniWarrior — APPROVED** [review]
Good work in adding in these placeholders. This will be a good foundation when this component is ready to be worked on.

**Review by milklot — APPROVED** [review]
All good, thank you for the fix

**Pull request merged**